src/libcore/intrinsics.rs

```diff
@@ -1523,4 +1523,7 @@ extern "rust-intrinsic" {
     /// Emits a `!nontemporal` store according to LLVM (see their docs).
     /// Probably will never become stable.
     pub fn nontemporal_store<T>(ptr: *mut T, val: T);
+
+    #[cfg(not(stage0))]
+    pub fn freeze<T>(ptr: *mut T, count: usize);
```

Comment:
We customarily add a doc comment referring the user to the libcore/ptr.rs version so we should do that here as well...

Reply:
Done